### ISSUE TITLE
"More" modals accessibility

### ISSUE THREAD

**Issue by tmacmath** [yalelibrary/YUL-DC#1780]
This is an overview of styling changes that should be made in the "More" modals (facet links)to enhance legibility/readability and improve color contrast. 

Modal Body:
- [x] Change font size of facet links to 14px.

- [x] Change item count font size to 14px.

Modal footer:

- [ ] ~Change font size of sort buttons to 14px.~

- [x] Give the first sort button, A-Z sort, a margin-right of 10px so that the focus outlines do not get cut off or overlap.

- [x] Button background color: make darker, as there is not enough contrast between the button color and the white background. The color I used for this mockup was #e1dfdf; but I need to do a bit more testing or for an approved color in the application $medium_grey.

- [x] Add arrows to the Previous and Next buttons (&laquo;Previous and Next&raquo;). I can provide the HTML for this.
     - [x] <<  and >> ought to be centered to the text, similar to the styling in the rest of the code. 

See the mockup below for reference. (Each of these changes can be put into separate tickets if necessary.)

![Screen Shot 2021-11-30 at 12.19.16 PM.png](https://images.zenhubusercontent.com/604a2edec7bb2836333e6ec0/b7262675-fa1d-4df1-8955-833018eefd32)

**Comment by tmacmath** [comment]
Links within the modal itself need a unique ID so that the font size of the facet links does not change.

**Comment by FCRodriguez7** [comment]
I used the double arrows in the image and the text double arrows.

**Comment by FCRodriguez7** [comment]
Im closer to completing the ticket: 

![image.png](https://images.zenhubusercontent.com/5e6013c029e314c92afd4769/5f6b0c98-1cb3-415e-af07-bb0ce5757b90)

**Comment by FCRodriguez7** [comment]
![image.png](https://images.zenhubusercontent.com/5e6013c029e314c92afd4769/9228243d-a9f9-40ed-aeaa-252a11d84b52)

**Comment by FCRodriguez7** [comment]
The PR https://github.com/yalelibrary/yul-dc-blacklight/pull/733

**Comment by FCRodriguez7** [comment]
Released blacklight v1.59.2

**Comment by mikeapp** [comment]
I'm seeing numbers (circled) in the background on UAT but not on production:
![Screen Shot 2021-12-17 at 3.56.14 PM.png](https://images.zenhubusercontent.com/5e5d472f410278efd81466ab/d49b5c65-988b-4244-8888-15d6eb6e279b)

**Comment by FCRodriguez7** [comment]
The numbers appearing are from the copy cut and dump.  They are the code line numbers from the original blacklight project code.   I removed them from the code.   

![image.png](https://images.zenhubusercontent.com/5e6013c029e314c92afd4769/f981582b-ab4a-4b60-a644-56d05366ebd3)

**Comment by FCRodriguez7** [comment]
![Screen Shot 2021-12-18 at 11.30.04 AM.png](https://images.zenhubusercontent.com/5e6013c029e314c92afd4769/9781da6b-765c-4843-8263-72b0c1cd7a21)

**Comment by FCRodriguez7** [comment]
@mikeapp I created a PR for the fix: https://github.com/yalelibrary/yul-dc-blacklight/pull/735

**Comment by FCRodriguez7** [comment]
Released blacklight v1.59.3

**Comment by motropuk** [comment]
Looks good on UAT
